### ISSUE TITLE
Component mutates content

### ISSUE THREAD

**Issue by andrewbranch** [12StarsMedia/ember-ui-sortable#2]
Hey there,

I love the easy-to-use API of this component, but I just noticed that the `content` array is mutated when a move occurs. In the spirit of data-down/actions-up, it would be great at least to have an option where the content array is not touched, which would allow the implementer to modify or replace (or conditionally do nothing to) the content array during the `move` action.

This is a feature that I need, so I’m about to fork and see if I can quickly get something working; are you open to a PR for this?

**Comment by ToddSmithSalter** [comment]
Absolutely.

**Comment by andrewbranch** [comment]
I'm having trouble running the tests. Dependencies install, the project builds, but then I get

```
Error: Assertion Failed: The tests file was not loaded. Make sure your tests index.ht
ml includes "assets/tests.js". at http://localhost:7357/assets/vendor.js, line 16745
Error: Could not find module `ember/resolver` imported from `dummy/tests/helpers/reso
lver` at http://localhost:7357/assets/vendor.js, line 146
```

and no tests run. Any ideas? Can you confirm a fresh install works for you?

**Comment by ToddSmithSalter** [comment]
@andrewbranch Odd. My tests are hanging without any errors on REPL. Digging into it.

**Comment by ToddSmithSalter** [comment]
Sorry, @andrewbranch. I can't quickly find a fix to get test running. I'll have to take a look another time.

**Comment by nickschot** [comment]
@andrewbranch is your implementation still available somewhere? Else I'll implement something similar. We need some kind of API for this as sortable changing the DOM (at least on "stop") is not ideal (ember should do this imo).

**Comment by andrewbranch** [comment]
@nickschot here's my fork: https://github.com/andrewbranch/ember-ui-sortable

**Comment by nickschot** [comment]
@andrewbranch thanks. This is a good option if we want to be backwards compatible (no clue if this is really necessary or how much ember-ui-sortable is actually used). 

I have already removed the content manipulation from my fork/draggable branch.

**Comment by ToddSmithSalter** [comment]
Addressed in release 0.3.1 #4
